android/java/proguard.flags

```diff
@@ -20,3 +20,7 @@
 -keep class org.chromium.chrome.browser.ChromeTabbedActivity {
     *** hideOverview(...);
 }
+
```

Comment:
Do we need this one? We use these flags only when `BraveReflectionUtil.InvokeMethod` is used.

Reply:
not sure. if `changeMethodOwner` works without this, then it's not needed. will check it now.

Reply:
thanks! removed proguard changes.